components/typography/typography.css

```diff
@@ -0,0 +1,56 @@
+h1,
+.h1 {
+  font-size: 3.2rem;
+  margin: 10px 0;
+}
+h2,
+.h1 {
+  font-size: 2.4rem;
+  margin: 10px 0;
+}
+h3,
+.h3 {
+  font-size: 1.6rem;
+  margin: 10px 0;
+}
+h4,
+.h4 {
+  font-size: 1.2rem;
+  margin: 10px 0;
+}
+
+h5,
+.h5 {
+  font-size: 8px;
+  margin: 10px 0;
+}
+p {
+  margin: 5px 0;
+}
+.p-xlg {
+  font-size: 1.8rem;
+}
+.p-lg {
+  font-size: 1.6rem;
+}
+.p-md {
+  font-size: 1.2rem;
+}
+.p-sm {
+  font-size: 8px;
+}
+.p-xsm {
+  font-size: 5px;
+}
+.text-wt-lt {
+  font-weight: 300;
+}
+.text-wt-md {
+  font-weight: 500;
+}
+.text-wt-bold {
```

Comment:
Refer to the Figma file, you can use the same font weight class names there
`txt-light, txt-md, txt-bold` etc

Reply:
Thanks for suggestion